pkg/fqdn/restore/restore.go

```diff
@@ -48,10 +48,12 @@ func (r IPRules) Sort() IPRules {
 // Sort is only used for testing
 // Sorts in place, but returns DNSRules for convenience
 func (r DNSRules) Sort() DNSRules {
-	for port, ipRules := range r {
-		if len(ipRules) > 0 {
-			ipRules = ipRules.Sort()
-			r[port] = ipRules
+	for proto, portMap := range r {
```

Comment:
Not directly related to this PR, but if this should be used only for testing, we should consider adding a `*testing.T` parameter to its signature, in order to make it explicit that this is intended only for testing purposes (same for `IPRules.Sort`).

Reply:
Done.